pokecli.py

```diff
@@ -64,7 +64,7 @@
 try:
     import pkg_resources
     pgoapi_version = pkg_resources.get_distribution("pgoapi").version
-    if pgoapi_version < '1.1.8':
+    if pgoapi_version != '1.1.6':
```

Comment:
Please help me understand... you are suggesting to change the API version requirements from 1.1.8 to 1.1.6?... Does not sound right to me.